packages/lesswrong/server/resolvers/moderationResolvers.ts

```diff
@@ -26,7 +26,7 @@ defineQuery({
     const loginEvents = await LWEvents.find({
       name: "login",
       "properties.ip": ipAddress,
-    }, {limit: 20}).fetch();
+    }, {limit: 100}).fetch();
```

Comment:
tempted to suggest that we include a projection here to only fetch the ip & userId, though it is a bit of a micro-optimization.